core-ui/src/components/LabelSelectorInput/LabelSelectorInput.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import './LabelSelectorInput.scss';
+import { Token } from 'fundamental-react/Token';
+
+//TODO: move this component to a shared "place"
+
+const labelRegexp = /[a-z0-9A-Z-_.]+=[a-z0-9A-Z-_.]+/;
```

Comment:
i believe it is not right, as `examplelabel=` is a proper label in kubernetes world, and does not match this regular expression. see: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set

Reply:
Should be fixed now.

Reply:
```suggestion
const labelRegexp = /[a-z0-9A-Z-_.]+=[a-z0-9A-Z-_.]*/;
```
still, it doesn't accept `something=` :( i believe the `+` is a problem here. take a look at my suggestion